clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -3607,6 +3648,11 @@ public void testIncrementalAlterConfigs()  throws Exception {
                     .setResourceType(ConfigResource.Type.BROKER.id())
                     .setErrorCode(Errors.NONE.code())
                     .setErrorMessage(ApiError.NONE.message()));
+            responseData.responses().add(new AlterConfigsResourceResponse()
```

Comment:
Is the presence of this response being checked anywhere?

Reply:
i'll add a get call there in the end